vbezbakh/docker/vbezbakh:devopscampsummer2019_vbezbakh_docker_example10/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:jessie
+
+RUN apt-get update && apt-get install -y \
+      build-essential \
+      zlib1g-dev \
+      libssl-dev \
+      libreadline6-dev \
+      libyaml-dev
+
+ADD http://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.1.tar.gz /tmp/
+RUN cd /tmp && \
+      tar -xzf ruby-2.1.1.tar.gz && \
+      cd ruby-2.1.1 && \
+      ./configure && \
+      make && \
+      make install && \
+      cd .. && \
+      rm -rf ruby-2.1.1 && \
+      rm -f ruby-2.1.1.tar.gz
```

Comment:
Seems, this dockerfile is not finished. Would be good to split it to build and execution parts and use 'COPY --from=builder ...' command to move built results into the second part/image.

Without it '*.tar.gz' archive (in the ADD command) will be stored in the image and your removal won't help to free disk space.